### PR TITLE
Update the Home page

### PR DESCRIPTION
**Pull request by hoffwm**
This commit makes changes and updates the home page of the
documentation site. refs#69

### CODE REVIEW

**Comment by hoffwm** [comment]
@aeslaughter @colejust The three columns of text are obviously not complete. Justin, if you would look at that and add comments for what you would like those to say, I will make the changes and add them. Any of the text can be changed so let me know if you want changes.

File: doc/content/index.md [hunk 1]
```diff
@@ -1,10 +1,127 @@
...
+  </div>
+  <div class="col s12 m4">
+    <div class="icon-block">
+      <h2 class="center black-text">
+        <i class="material-icons">
+          check_box
+        </i>
+      </h2>
+      <h5 class="center">
+          Stochastic Simulations
```
Comment (cbolisetti) [inline doc/content/index.md]: Can you change this to 'Risk Assessment'?

**Comment by hoffwm** [comment]
@colejust @cbolisetti @aeslaughter I made those changes, click the link to view the site. I really think the three columns of text in the middle should be about the same length, so I would say the first one needs to be shortened or the second two need to be expanded. Any thoughts?

**Comment by aeslaughter** [comment]
@hoffwm I do think the text columns need to be better aligned.

**Comment by hoffwm** [comment]
@aeslaughter agreed. @colejust told me just to get it merged in and it can be fixed later. If anyone has ideas on how to shorten the one or lengthen the others I will make the changes.... I just can't really think of any changes to make...

**Comment by colejust** [comment]
@aeslaughter when I approve it does not remove the "Review required"...why?

**Comment by aeslaughter** [comment]
@colejust This is a limitation to GitHub, which is based on the level of permissions you have.

**Pull request merged**